src/cmd/add.nim

```diff
@@ -81,11 +82,31 @@ proc paxAdd*(input: string, strategy: string): void =
   echoMod(cfMod, moreInfo = true)
   echo ""
 
-  returnIfNot promptYN("Are you sure you want to install this mod?", default = true)
+  returnIfNot promptYN("Are you sure you want to install this mod?",
+      default = true)
 
   echoInfo "Installing ", cfMod.name.cyanFg, ".."
   manifest.installMod(cfMod.projectId, cfModFile.fileId)
 
+  echoInfo "Resolving Dependencies..."
+  ## Get Dependendencies
+  if len(cfModFile.dependencies) > 0:
+    if promptYN("Do you want to install dependencies?", default = true):
```

Comment:
In my opinion, since the users expect installed mods to work, it would be best to install dependencies by default and add a `--no-deps` parameter that would not add/remove dependencies - for the really uncommon usecases.